src/plugins/session-timeout/session-timeout.js

```diff
@@ -249,13 +250,13 @@ var $modal, $modalLink, countdownInterval, i18n, i18nText,
 		// Clear the keepalive timeout to avoid double firing of requests
 		clearInterval( $( event.target ).data( keepaliveEvent ) );
 
+		$buttonEnd = $( buttonStart + confirmClass + " btn btn-default'>" +
+			i18nText.buttonEnd + buttonEnd )
+			.data( "logouturl", settings.logouturl );
```

Comment:
After implementing your suggested change:
```
window.location.href = settings.signInUrl ? settings.signInUrl : settings.logouturl;
```
moving the code is not really necessary now.  I originally moved the code so that I could assign the `signInUrl` to the `logouturl` before adding the settings to the `data` attribute of `$buttonContinue`.  Should I change it back?

Reply:
@sgdowney Yes, please move it back. Once it's done I can approve the PR.